File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    lux, luy = 51, 51
    rdx, rdy = 0, 0
    
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == '#':
                if i < lux:
                    lux = i
                if j < luy:
                    luy = j
                if rdx < i + 1:
                    rdx = i + 1
                if rdy < j + 1:
                    rdy = j + 1
        
    
    return [lux, luy, rdx, rdy]

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int lux = Integer.MAX_VALUE, luy = Integer.MAX_VALUE;
        int rdx = 0, rdy = 0;
        
        for(int i = 0; i < wallpaper.length; i++){
            for(int j = 0; j < wallpaper[i].length(); j++){
                char temp = wallpaper[i].charAt(j);
                if(temp == '#'){
                    lux = Math.min(lux, i);
                    luy = Math.min(luy, j);
                    rdx = Math.max(rdx, i);
                    rdy = Math.max(rdy, j);
                }
            }
        }
        
        
        int[] answer = {lux, luy, rdx + 1, rdy + 1};
        return answer;
    }
}